cdc/processor/pipeline/sorter.go

```diff
@@ -211,8 +220,28 @@ func (n *sorterNode) Receive(ctx pipeline.NodeContext) error {
 					zap.Uint64("oldResolvedTs", oldResolvedTs))
 			}
 			atomic.StoreUint64(&n.resolvedTs, rawKV.CRTs)
+
+			if resolvedTs > n.barrierTs &&
+				!redo.IsConsistentEnabled(n.replConfig.Consistent.Level) {
+				// Do not send resolved ts events that is larger than
+				// barrier ts.
+				// When DDL puller stall, it may cause data pile up in memory,
```

Comment:
I think it is a bit confusing that what this "it" refers to. 
Do you mean `If we send a resolved ts larger than barrier ts` ?